flutter-idea/src/io/flutter/analytics/Analytics.java

```diff
@@ -286,5 +283,33 @@ public void send(String url, Map<String, String> values) {
         }
       });
     }
+
+    byte[] createPostData(@NotNull Map<String, String> values) throws UnsupportedEncodingException {
+      final StringBuilder postData = new StringBuilder();
+      for (Map.Entry<String, String> param : values.entrySet()) {
+        if (postData.length() != 0) {
+          postData.append('&');
+        }
+        postData.append(URLEncoder.encode(param.getKey(), "UTF-8"));
+        postData.append('=');
+        postData.append(URLEncoder.encode(param.getValue(), "UTF-8"));
+      }
+      return postData.toString().getBytes(StandardCharsets.UTF_8);
+    }
+  }
+
+  private static class NonTransport extends HttpTransport {
```

Comment:
Do you mean you will temporarily change code to use this `NonTransport` class when you're debugging analytics code locally? Can you add a comment about that intention here?

Reply:
Done.